resources/benchmark-runner.mjs

```diff
@@ -5,24 +5,29 @@ export class BenchmarkTestStep {
     }
 }
 
-class BenchmarkState {
-    constructor(suites) {
+class BenchmarkState
```

Comment:
Oh, class declaration puts { on the same line. Just function declaration that puts { on a new line.
Sorry, should have been more clear.